plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -2299,6 +2329,10 @@ public void finishStatisticsCollection(ConnectorSession session, ConnectorTableH
     {
         DeltaLakeTableHandle tableHandle = (DeltaLakeTableHandle) table;
         AnalyzeHandle analyzeHandle = tableHandle.getAnalyzeHandle().orElseThrow(() -> new IllegalArgumentException("analyzeHandle not set"));
+        if (analyzeHandle.isInitialAnalyze()) {
+            generateMissingFileStatistics(session, tableHandle, computedStatistics);
```

Comment:
Why the condition?
With incremental analyze we could do this as well. It's just that we would fill min/max for subset of files only.

(we need to revisit https://github.com/trinodb/trino/blob/4a6c08a5c3535f4da9dddee20b288d1c1a9833e6/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java#L163-L165 as well.
that code assumes ANALYZE covers data only but now it became aware of file boundaries)

Reply:
I would like to exclude incremental `ANALYZE` as separate PR if that's ok.

Reply:
Can we leave a TODO comment?